packages/react/src/contexts/theme.test.tsx

```diff
@@ -63,8 +71,8 @@ test('it provides toggleTheme functionality', () => {
 });
 
 test('handles mutations', () => {
-  mountWrapper();
-  expect(observe.called).toBe(true);
+  renderProvider();
+  expect(observe).toHaveBeenCalled;
```

Comment:
Don't we need to invoke this?

```suggestion
  expect(observe).toHaveBeenCalled();
```